packages/flutter/test/rendering/object_test.dart

```diff
@@ -157,6 +157,35 @@ void main() {
     expect(() => renderObject1.getTransformTo(renderObject2), throwsAssertionError);
   });
 
+  test('RenderObject.computeTransformToRenderObject works for siblings and descendants', () {
+    final PipelineOwner owner = PipelineOwner();
+    final TestRenderObject renderObject1 = TestRenderObject()..attach(owner);
+    final TestRenderObject renderObject11 = TestRenderObject();
+    final TestRenderObject renderObject12 = TestRenderObject();
+
+    renderObject1
+      ..add(renderObject11)
+      ..add(renderObject12);
+    expect(renderObject11.computeTransformToRenderObject(renderObject12), equals(Matrix4.identity()));
+    expect(renderObject1.computeTransformToRenderObject(renderObject11), equals(Matrix4.identity()));
+    expect(renderObject1.computeTransformToRenderObject(renderObject12), equals(Matrix4.identity()));
+    expect(renderObject11.computeTransformToRenderObject(renderObject1), equals(Matrix4.identity()));
+    expect(renderObject12.computeTransformToRenderObject(renderObject1), equals(Matrix4.identity()));
+
+    expect(renderObject1.computeTransformToRenderObject(renderObject1), equals(Matrix4.identity()));
+    expect(renderObject11.computeTransformToRenderObject(renderObject11), equals(Matrix4.identity()));
+    expect(renderObject12.computeTransformToRenderObject(renderObject12), equals(Matrix4.identity()));
+  });
```

Comment:
Can we add a test that actually compute something meaningful to make sure the implementation actually does what it promises? I could satisfy all these tests by "accidentally" replacing the implementation of `computeTransformToRenderObject` with:

```dart
Matrix4 computeTransformToRenderObject(...) => Matrix4.identity();
```